src/lib/system/datasources/moor_database.dart

```diff
@@ -240,7 +240,12 @@ class MoorDatabase extends _$MoorDatabase {
               const KeyValueEntriesCompanion(key: Value(PERSISTENT_SHUFFLEMODE), value: Value('0')),
             );
             await into(keyValueEntries).insert(
-              const KeyValueEntriesCompanion(key: Value(SETTING_ALLOWED_EXTENSIONS), value: Value(ALLOWED_FILE_EXTENSIONS)),
+              const KeyValueEntriesCompanion(
+                  key: Value(SETTING_ALLOWED_EXTENSIONS), value: Value(ALLOWED_FILE_EXTENSIONS)),
+            );
+            await into(keyValueEntries).insert(
+              const KeyValueEntriesCompanion(
+                  key: Value(SETTING_ALLOWED_EXTENSIONS), value: Value('false')),
```

Comment:
```suggestion
                  key: Value(SETTING_PLAY_ALBUMS_IN_ORDER), value: Value('false')),
```
Small copy paste error here.